requirements.txt

```diff
@@ -1,2 +1,3 @@
 pyjq==2.1.0
 netaddr==0.7.19
+six>=1.10.0
```

Comment:
Lock this version as well.

Reply:
Would you accept: `six<2,>=1.10.0`?  Which would lock it to a *Major* version of `1.`

See above for justification and ask for clarification.